RoeiJeRot.Logic/PasswordHasher.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Text;
+
+namespace RoeiJeRot.Logic
+{
+    class PasswordHasher
+    {
+        public bool CompareHash(string dbHash, string password)
+        {
+            bool value = false;
+
+            if (dbHash == GetPasswordHash(password))
+            {
+                value = true;
+            }
+
+            return value;
+        }
+
+        public string GetPasswordHash(string password)
```

Comment:
Kan dit hernoem dit Naar `Hash`

Kan worden aangeroepen als: `hasher.Hash("123")`.
Is korter en omschrijf de actie die het doet.

Reply:
En de klasse kan in principe een static klasse worden